tapscript/send.go

```diff
@@ -648,9 +648,14 @@ func SignVirtualTransaction(vPkt *tappsbt.VPacket, signer Signer,
 		prevAssets[input.PrevID] = input.Asset()
 	}
 
+	assetOutputs := make([]*asset.Asset, len(outputs))
```

Comment:
nit: could make use of the `fn.Map()` function here. Doesn't really make it shorter but makes review a bit easier IMO (since creation of the slice with `make` is abstracted away).